### PR TITLE
Feature/adbp 1480

### PR DESCRIPTION
**Pull request by muh-techie**
### Why

Please give a short description why these code changes are being made, by replacing this line.

Link to Jira ticket: https://101digital.atlassian.net/browse/ADBP-1480

### ToDo

- [ ]
- [ ]
- [ ]

If this is a `Draft:` or `WIP:` PR, please describe still needed actions, else delete section

### CODE REVIEW

File: src/components/adb-account-openning/index.tsx [hunk 1]
```diff
@@ -34,41 +47,74 @@ const ADBAccountOpeningComponent = (props: ADBAccountOpeningProps) => {
...
+          {benefits?.map((row) => (
+            <View key={row} style={styles.row}>
+              <ImageIcon size={25} />
+              <Text style={styles.rowTitle}>{row}</Text>
+            </View>
+          ))}
+          <View style={styles.policyContainer}>
+            <ShieldIcon size={20} />
+            <Text style={styles.policyText}>
+              By continuing, I agree and comply with {'{ADB}’s'}{' '}
```
Comment (sharmal33) [inline src/components/adb-account-openning/index.tsx:91]: move this to translation file

File: src/components/adb-account-openning/index.tsx [hunk 2]
```diff
@@ -34,41 +47,74 @@ const ADBAccountOpeningComponent = (props: ADBAccountOpeningProps) => {
...
+              <TouchableOpacity>
+                <Text style={styles.policyTextLinks} onPress={handleOnPolicyPress}>
+                  privacy policy
```
Comment (sharmal33) [inline src/components/adb-account-openning/index.tsx:94]: plz move all string to translation file

File: src/context/onboarding-context.ts [hunk 3]
```diff
@@ -689,34 +705,64 @@ export function useAccountOriginationContextValue(): AccountOriginationContextDa
...
+    }, []);
+  
+    const clearPassword = () => savePassword(undefined);
+    const getUserPIN = async () => {
+      const value = await SInfo.getItem("userPIN", {
+        sharedPreferencesName: "mySharedPrefs",
+        keychainService: "myKeychain",
+      });
+      setPIN(value);
+      console.log("getPIN -> value", value);
```
Comment (sharmal33) [inline src/context/onboarding-context.ts:729]: remove logs

File: src/service/onboarding-service.ts [hunk 4]
```diff
@@ -922,4 +924,19 @@ export class AccountOriginationService {
...
+
+  public getContentTemplate = async (templateName: string) => {
+    if (this._contentTemplateClient) {
+      const response = await this._contentTemplateClient.post('/contents', {
+        data: {},
+        documentFormat: 'HTML',
+        templateName,
+        entityId: 'ADB',
+        appId: 'SYSTEM',
+      });
```
Comment (sharmal33) [inline src/service/onboarding-service.ts:936]: why values are hardcoded

**Review by sharmal33 — CHANGES_REQUESTED** [review]

File: src/context/onboarding-context.ts (hunk 3 above)
Comment (muh-techie) [reply src/context/onboarding-context.ts:729]: @sharmal33 its a old code but I have remove it

File: src/service/onboarding-service.ts (hunk 4 above)
Comment (muh-techie) [reply src/service/onboarding-service.ts:936]: its need to be hardcode as Tung mention

image

**Review by sharmal33 — APPROVED** [review]

**Pull request merged**